src/app/store/accounts/blockchain/bitcoin/bitcoin-signer.ts

```diff
@@ -105,6 +105,7 @@ interface CreateSignersForAllNetworkTypesArgs {
   paymentFn: (keychain: HDKey, network: BitcoinNetworkModes) => unknown;
   mainnetKeychainFn: (accountIndex: number) => BitcoinAccount | undefined;
   testnetKeychainFn: (accountIndex: number) => BitcoinAccount | undefined;
+  regtestKeychainFn: (accountIndex: number) => BitcoinAccount | undefined;
```

Comment:
Don't understand these changes and don't think they're needed? Afaik, regtest uses the same derivation path, and thus keychain as testnet?

Reply:
Yeah, tried without this, it works just fine, so I'll revert this. At first, even though I used regtest, testnet addresses were shown, thought maybe this is one of the reasons, but turns out, it's not. Thank you!